editor/action_map_editor.cpp

```diff
@@ -214,9 +216,15 @@ void InputEventConfigurationDialog::_listen_window_input(const Ref<InputEvent> &
 		return;
 	}
 
-	// Ignore mouse
-	Ref<InputEventMouse> m = p_event;
-	if (m.is_valid()) {
+	// Ignore mouse motion
+	Ref<InputEventMouseMotion> mm = p_event;
+	if (mm.is_valid()) {
+		return;
+	}
+
+	// Ignore mouse button if not in the detection rect
+	Ref<InputEventMouseButton> mb = p_event;
+	if (mb.is_valid() && !mouse_over_detection_rect) {
```

Comment:
Here is what I meant in my comment
`! Color_rect->get_rect().contains(get_mouse_pos()) `

Reply:
Yes ! I will try this way.

Reply:
Wow, i had to struggled quite a lot with relative / absolute position of mouse / base window and UI elements.

I've finished to find a way that seems to work well in all situation : 

```
		Rect2 r = mouse_detection_rect->get_rect();
		r.set_position(mouse_detection_rect->get_screen_position());
		if (!r.has_point(get_base_window()->get_position() + get_mouse_position())) {
			return;
		}
```